src/main/java/frc/robot/Robot.java

```diff
@@ -54,11 +55,13 @@ public class Robot extends TimedRobot {
   private Climber climber;
   private Turret turret;
   private Hood hood;
+  public int TRL;
```

Comment:
Given that you named your variable as an acronym, you should put a comment that indicates:
A) What TRL stands for, and
B) how TRL is meant to be used in the code

Be brief, but informative

Also, more importantly, it seems like TRL should only ever be set to 1 or 0, if this is the case, TRL should be a boolean value